### PR TITLE
[1LP][RFR] varmeth

### PR DESCRIPTION
**Pull request by digitronik**
## Purpose or Intent

_varmeth_ is one of best solution of context switching. created one small library in `RedHatQE` org. so other project in RedHat can take advantage of it.

https://github.com/RedHatQE/varmeth

### CODE REVIEW

File: requirements/frozen_docs.py3.txt [hunk 1]
```diff
@@ -265,6 +265,7 @@ tabulate==0.7.7
 tzlocal==1.5.1
 uritemplate==3.0.0
 urllib3==1.24.2
+varmeth==0.0.2
```
Comment (john-dupuy) [inline requirements/frozen_docs.py3.txt:268]: @digitronik this package should also be added to `template_scanned_imports.py`, cf `miq requirement add varmeth`

Comment (digitronik) [reply requirements/frozen_docs.py3.txt:268]: right

**Comment by RonnyPfannschmidt** [comment]
I hoped we could have killed this one for sentaku

**Comment by mshriver** [comment]
@RonnyPfannschmidt sentaku is certainly more powerful, but there's still some spots where we're using varmeth.

**Comment by digitronik** [comment]
@RonnyPfannschmidt yup, I love both `sentaku` and `varmeth`.
Personally, i'm perfering `sentaku` when project rich with multiple `context` (eg. project has good support of rest and ui).

I'm considering `varmeth` as more general solution for poor context support. even its usefull to me in daily hacks like delete operation from two differnt pages delete(method='All', delete(method='summary').

**Comment by RonnyPfannschmidt** [comment]
If method is not picked dynamically, wouldn't real method names indicate better?

**Pull request merged**